k8s/generate_code.sh

```diff
@@ -75,6 +70,12 @@ if [ -e ${GOPATH}/src/${LH_MANAGER_DIR}/k8s/patches/crd ]; then
 fi
 popd
 
-echo "# Generated by the CRDs from ${APIS_DIR}" > ${GOPATH}/src/${LH_MANAGER_DIR}/k8s/crds.yaml
+echo "Generating crds.yaml from ${APIS_DIR}"
+rm -rf ${GOPATH}/src/${LH_MANAGER_DIR}/k8s/crds.yaml
 kustomize build ${CRDS_DIR} >> ${GOPATH}/src/${LH_MANAGER_DIR}/k8s/crds.yaml
 rm -r ${CRDS_DIR}
+
+# Replace labels and namespace in crds.yaml. The crds.yaml is used by helm chart and used for generating the installation manifests.
```

Comment:
Suggest moving this to the action (you can create another script used in the action or even embed it), since the newly updated crds.yaml including helm-template becomes useless here if we still want to keep crds.yaml in this repo. I know saving it here is for development/testing purposes.